kubernetes_test.go

```diff
@@ -373,4 +373,46 @@ func TestTransformRoleMappings(t *testing.T) {
 			t.Errorf("TransformRoleMappings() returned unexpected object: %+v, want %+v", got, want)
 		}
 	})
+
+	// Test getting the instance IAM role
+	t.Run("Translate permissionSet name to role ARN", func(t *testing.T) {
+		mappings := []SSORoleMapping{
+			{
+				RoleARN:       "arn:aws:iam::123456789012:role/AWSReservedSSO_devops_0123456789abcdef",
+				PermissionSet: "",
+				Username:      "",
+				Groups:        []string{},
+			},
+		}
+
+		roles := []types.Role{
+			{
+				RoleName: aws.String("AWSReservedSSO_devops_0123456789abcdef"),
+				Path:     aws.String("/aws-reserved/sso.amazonaws.com/eu-west-1/"),
+				Arn:      aws.String("arn:aws:iam::123456789012:role/aws-reserved/sso.amazonaws.com/eu-west-1/AWSReservedSSO_devops_0123456789abcdef"),
+			},
+		}
+
+		want := []SSORoleMapping{
+			{
+				RoleARN:       "arn:aws:iam::123456789012:role/AWSReservedSSO_devops_0123456789abcdef",
+				PermissionSet: "",
+				Username:      "",
+				Groups:        []string{},
+			},
+			{
+				RoleARN:       "arn:aws:iam::123456789012:role/node-role",
+				PermissionSet: "",
+				Username:      "",
+				Groups:        []string{},
+			},
+		}
+		iamRoleARN := "arn:aws:iam::" + "123456789012" + ":role/" + getInstanceRole()
+		mappings = addWorkerNodeRoleBindings(mappings, iamRoleARN)
```

Comment:
to test `getInstanceRole()`, we would need to mock aws-sdk lib, though to simplify it, maybe we could just do:
```suggestion
		iamRoleARN := "arn:aws:iam::123456789012:role/node-role"
		mappings = addWorkerNodeRoleBindings(mappings, iamRoleARN)
```

Reply:
How about that instead?